.env.example

```diff
@@ -0,0 +1,75 @@
+APP_NAME=interns2025c
+VITE_APP_NAME=interns2025c
+APP_ENV=local
+APP_KEY=base64:sCsJw8z+d/4ymp0OvzSip2h4Vp2hZZhpV2uOxgTqP94=
+APP_DEBUG=true
+APP_URL=http://interns2025c.blumilk.local.env
+
+APP_LOCALE=pl
+APP_FALLBACK_LOCALE=pl
+APP_FAKER_LOCALE=pl
+
+APP_MAINTENANCE_DRIVER=file
+APP_MAINTENANCE_STORE=database
+
+BCRYPT_ROUNDS=12
+
+LOG_CHANNEL=stack
+LOG_DEPRECATIONS_CHANNEL=null
+LOG_LEVEL=debug
+
+DB_CONNECTION=pgsql
+DB_HOST=interns2025c-db-dev
+DB_PORT=5432
+DB_DATABASE=interns2025c
+DB_USERNAME=interns2025c
+DB_PASSWORD=password
+DB_ROOT_PASSWORD=example
+
+SESSION_DRIVER=redis
+SESSION_LIFETIME=120
+SESSION_ENCRYPT=false
+SESSION_PATH=/
+SESSION_DOMAIN=null
+
+BROADCAST_CONNECTION=log
+FILESYSTEM_DISK=local
+QUEUE_CONNECTION=redis
+
+CACHE_STORE=redis
+CACHE_PREFIX=
+
+MEMCACHED_HOST=127.0.0.1
+
+REDIS_CLIENT=phpredis
+REDIS_HOST=interns2025c-redis-dev
+REDIS_PASSWORD=null
+REDIS_PORT=6379
+
+MAIL_MAILER=smtp
+MAIL_HOST=mailpit
+MAIL_PORT=1025
+MAIL_USERNAME=null
+MAIL_PASSWORD=null
+MAIL_ENCRYPTION=null
+MAIL_FROM_ADDRESS="hello@example.com"
+MAIL_FROM_NAME="${APP_NAME}"
+
+AWS_ACCESS_KEY_ID=
+AWS_SECRET_ACCESS_KEY=
+AWS_DEFAULT_REGION=us-east-1
+AWS_BUCKET=
+AWS_USE_PATH_STYLE_ENDPOINT=false
+
+# DOCKER
+DOCKER_APP_HOST_PORT=63851
+DOCKER_DATABASE_HOST_PORT=63853
+DOCKER_MAILPIT_DASHBOARD_HOST_PORT=63854
+DOCKER_REDIS_HOST_PORT=63852
+DOCKER_INSTALL_XDEBUG=true
```

Comment:
Are u sure about using that ports?. Let's change that - its default configuration from boilerplate repo.